### PR TITLE
Clipboard failure behavior

### PR DESCRIPTION
**Pull request by maxjacobson**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [root]
 name = "rooster"
-version = "2.3.0"
+version = "2.4.0"
```
Comment (maxjacobson) [inline Cargo.lock]: (this changed when I ran `cargo build` -- kept it into a separate commit, but it seems correct given that it matches `Cargo.toml`, right?)

**Comment by conradkleinespel** [comment]
@maxjacobson Looking good ! You rock ! Thanks so much for your contribution !

Would you mind rephrasing "We tried" to "I tried", just to stay consistent with the other info messages ? If you can't right now, that's fine, I'll do it later this week-end. Once that's done, I'll publish a new version, `2.5.0`, to crates.io :smiley:

File: Cargo.lock (hunk 1 above)
Comment (conradkleinespel) [inline Cargo.lock]: :+1:

**Comment by conradkleinespel** [comment]
@maxjacobson I've just published `v2.4.1` to crates.io and created a release on Github: https://github.com/conradkleinespel/rooster/releases/tag/v2.4.1

**Pull request merged**